compiler_tester/src/vm/eravm/system_context.rs

```diff
@@ -230,6 +230,35 @@ impl SystemContext {
         }
 
         if target == Target::EVM {
+            let padded_index = [[0u8; 16], 0_u128.to_be_bytes()].concat();
+            let padded_slot =
+                web3::types::H256::from_low_u64_be(Self::SYSTEM_CONTEXT_BLOCK_HASH_POSITION)
+                    .to_fixed_bytes()
+                    .to_vec();
+            let key = web3::signing::keccak256([padded_index, padded_slot].concat().as_slice());
+
+            let mut hash = web3::types::U256::from_str(match target {
+                Target::EraVM => Self::ZERO_BLOCK_HASH_ERAVM,
```

Comment:
Looks like this branch is nor reachable

Reply:
Done here [fc177bc](https://github.com/matter-labs/era-compiler-tester/pull/82/commits/fc177bcc086e83f9a1041fe6879ba498676e5b27)